api/src/migrations/geoprocessing/1611221157285-initialGeoDBSetup.ts

```diff
@@ -0,0 +1,185 @@
+import {MigrationInterface, QueryRunner} from "typeorm";
+
+export class initialGeoDBSetup1611221157285 implements MigrationInterface {
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+      await queryRunner.query(`
+      CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+      CREATE EXTENSION IF NOT EXISTS tablefunc;
+      CREATE EXTENSION IF NOT EXISTS plpgsql;
+      CREATE EXTENSION IF NOT EXISTS postgis;
+      CREATE EXTENSION IF NOT EXISTS postgis_raster; -- OPTIONAL
+      CREATE EXTENSION IF NOT EXISTS postgis_topology; -- OPTIONAL
```

Comment:
This needs wrapping in a guard in case this is ever run in pg < 13 - see https://github.com/Vizzuality/marxan-cloud/pull/23/files#diff-6771bdcda9d1d20d78abccfdd6dad042b632e7a1693145fba93697c5143eca2dR9